real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/ui/GraphPanel.java

```diff
@@ -1,49 +1,47 @@
 package com.roboclub.robobuggy.ui;
 
-import java.awt.Color;
-import java.awt.GridLayout;
-import javax.swing.BorderFactory;
-import javax.swing.JPanel;
+
 import com.roboclub.robobuggy.messages.SteeringMeasurement;
 import com.roboclub.robobuggy.ros.Message;
 import com.roboclub.robobuggy.ros.MessageListener;
-import com.roboclub.robobuggy.ros.SensorChannel;
+import com.roboclub.robobuggy.ros.NodeChannel;
 import com.roboclub.robobuggy.ros.Subscriber;
 
-public class GraphPanel extends JPanel {
+public class GraphPanel extends RoboBuggyGUIContainer {
 	private static final long serialVersionUID = -5453262887347328140L;
 
-	private AngleGraph steering;
-	private AngleGraph roll;
-	private AngleGraph pitch;
-	private AngleGraph yaw;
+	private AngleGraph steeringGraph;
+	private AngleGraph rollGraph;
+	private AngleGraph pitchGraph;
+	private AngleGraph yawGraph;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Perhaps 'yawGraph' could be replaced by a local variable.](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1757344775/issues/source?bid=2938665&fileBranchId=3059924#l16)